source/index.html.md

```diff
@@ -1850,14 +1850,314 @@ This endpoint gets a list of unsubscribed phone numbers.
+| custom_field[default_value]                 | string  | A default value for when none is provided. Must adhere to the `pattern` regex.         |
+
+## Deleting a Custom Field
+
+```ruby
+require 'net/http'
+require 'uri'
+
+uri = URI.parse('https://api.tatango.com/api/v2/lists/ID/custom_fields')
+http = Net::HTTP.new(uri.host, uri.port)
+request = Net:HTTP::Delete.new(uri.request_url)
+request.basic_auth("emailaddress@mydomain.com", "my_api_key")
+request.body(key: "some_key")
+response = http.request(request)
+```
+
+```shell
+curl "https://api.tatango.com/api/v2/lists/ID/custom_fields" -d '{"key": "some_key"}' -X DELETE \
+	-H "Accept: application/json" \
+	-H "Content-Type: application/json" \
+	-u emailaddress@mydomain.com:my_api_key \
+	-H "Host: example.org" \
+	-H "Cookie: "
+```
+
+```javascript
+var request = new XMLHttpRequest();
+request.open(
+  "DELETE",
+  "https://api.tatango.com/api/v2/lists/ID/custom_fields",
+  false
+);
+request.setRequestHeader(
+  "Authorization",
+  "Basic " + btoa("emailaddress@mydomain.com:my_api_key")
+);
+var data = JSON.stringify({
+  key: "some_key",
+});
+request.send(data);
+```
+
+> The above command returns JSON structured like this:
+
+```json
+{
+  "status": "Custom field successfully deleted"
+}
+```
+
+This endpoint deletes a custom field.
+
+### HTTP Request
+
+`DELETE https://api.tatango.com/api/v2/lists/ID/custom_fields`
+
+### URL Parameters
+
+| Parameter | Description    |
+| --------- | -------------- |
+| ID        | ID of the list |
+
+### JSON Parameters (JSON Object)
+
+| Key | Description          |
+| --- | -------------------- |
+| key | The custom field key |
+
+## Deleting a tag
+
+```ruby
+require 'net/http'
+require 'uri'
+
+uri = URI.parse('https://api.tatango.com/api/v2/lists/ID/tags')
+http = Net::HTTP.new(uri.host, uri.port)
+request = Net:HTTP::Delete.new(uri.request_url)
+request.basic_auth("emailaddress@mydomain.com", "my_api_key")
+request.body(tags: ["some_tag", "some_other_tag"])
+response = http.request(request)
+```
+
+```shell
+curl "https://api.tatango.com/api/v2/lists/ID/tags" -d '{"tags": ["some_tag", "some_other_tag"]}' -X DELETE \
+	-H "Accept: application/json" \
+	-H "Content-Type: application/json" \
+	-u emailaddress@mydomain.com:my_api_key \
+	-H "Host: example.org" \
+	-H "Cookie: "
+```
+
+```javascript
+var request = new XMLHttpRequest();
+request.open("DELETE", "https://api.tatango.com/api/v2/lists/ID/tags", false);
+request.setRequestHeader(
+  "Authorization",
+  "Basic " + btoa("emailaddress@mydomain.com:my_api_key")
+);
+var data = JSON.stringify({
+  tags: ["some_tag", "some_other_tag"],
+});
+request.send(data);
+```
+
+> The above command returns JSON structured like this:
+
+```json
+{
+  "status": "Tags enqueued for deletion"
+}
+```
+
+This endpoint deletes tags. After the endpoint is called, please allow up to 10 minutes for the tags provided to be removed from all subscribers.
```

Comment:
Nitpick but `This endpoint deletes tags from all subscribers with the tag.` Just want to be clear what it does up front.